docs/im/refresh-analyzer/index.md

```diff
@@ -1,8 +1,11 @@
 ---
 layout: default
 title: Refresh Search Analyzer
-parent: Index State Management
-nav_order: 6
+nav_order: 40
+parent: Index Management
+has_children: true
```

Comment:
I think this should be false. It's showing up with an expand arrow in the nav, but it has no children that I can see.